core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -908,6 +908,7 @@ class GroupCoordinator(val brokerId: Int,
    * @param offsetTopicPartitionId The partition we are now leading
    */
   def onElection(offsetTopicPartitionId: Int): Unit = {
+    info(s"Becoming the group coordinator for partition $offsetTopicPartitionId")
```

Comment:
It would also be nice to have the groups that are coordinated in the log message. I guess that is hardly possible, isn't it?

Reply:
Yeah that cannot be known until loading is completed.

Reply:
But during the loading process, we do have log entry as

```
Loading / Unloading group metadata for ...
```

per group.

Reply:
That is correct. Thank you for reminding me!